get_trust_anchor.py

```diff
@@ -344,13 +325,18 @@ def get_matching_ksk(KSKRecords, ValidTrustAnchors):
     return MatchedKSKs
 
 
-def export_ksk(ValidKSKs, DSRecordFileName, DNSKEYRecordFileName):
```

Comment:
The reason for having the filenames as params is that later on we may want to have a command line option to change the filename, please keep them.